packages/thirdweb/src/client/client.ts

```diff
@@ -47,6 +47,28 @@ type ClientOptions = Prettify<{
        */
       gatewayUrl?: string;
     };
+    baseUrls?: {
+      /**
+       * The base URL for the RPC server.
+       * @default "rpc.thirdweb.com"
+       */
+      rpc?: string;
+      /**
+       * The base URL for the embedded wallet service
+       * @default "embedded-wallet.thirdweb.com"
+       */
+      embeddedWallet?: string;
+      /**
+       * The base URL for the payment server.
+       * @default "interstate.thirdweb.com"
+       */
+      pay?: string;
+      /**
+       * The base URL for the storage server.
+       * @default "storage.thirdweb.com"
+       */
+      storage?: string;
```

Comment:
not sure if i like this as a public API. 

would this be used by anyone else but us?

I feel like it would be simpler to have an internal

setThirdwebDomain("prod" | "dev")

that sets a global domain var